Jenkinsfile

```diff
@@ -100,6 +100,23 @@ pipeline {
                 }
             }
         }
+        stage('Run OWASP ZAP security framework') {
+            agent {
+                docker {
+                    image 'owasp/zap2docker-stable'
+                    args '-u root:sudo --net demo -v ${WORKSPACE}:/zap/wrk:rw'
+                    reuseNode true
+                }
+            }
+            environment {
+                CONTAINER_HTTP_URL = "http://${APP_NAME}:8080"
```

Comment:
@RobertKelly @jknight-liatrio - Should we be hard coding the port?  Shouldn't that be a parameter that we pass in?

Reply:
This is specific for the app running in the docker container for testing. I modeled it after the selenium test step.